v0/pipeline/stages/cf/execution-wrapper-config.yaml

```diff
@@ -68,6 +68,7 @@ properties:
     - $ref: ../../steps/cd/serverless-aws-lambda-prepare-rollback-v2-step-node.yaml
     - $ref: ../../steps/cd/serverless-aws-lambda-rollback-v2-step-node.yaml
     - $ref: ../../steps/cd/aws-cdk-bootstrap-step-node.yaml
+    - $ref: ../../steps/cd/aws-cdk-synth-step-node.yaml
```

Comment:
https://github.com/harness/harness-schema/blob/main/v0/pipeline/stages/cd/execution-wrapper-config.yaml its missing to be added here